testplan/runners/pools/base.py

```diff
@@ -662,11 +679,12 @@ def aborting(self):
         """Aborting logic."""
         self.logger.debug('Aborting pool {}'.format(self))
 
-        super(Pool, self).stopping()  # stop the loop and the monitor
         for worker in self._workers:
             worker.abort()
+
+        super(Pool, self).stopping()  # stop the loop and the monitor
```

Comment:
This should call aborting() in the parent class not stopping() - we should not wait to join() the work loop when aborting.

Reply:
This hasn't been resolved?